.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Is this really meant to be part of the PR?

Reply:
As it, no. But yes, I plan to include helpers for most common IDE. But it's not yet ready (PR is still a draft)